src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -318,13 +323,20 @@
     default-so-linger-in-milliseconds))
 
 (schema/defn ^:always-validate
+  common-connector-config :- WebserverConnectorCommon
+  [config :- WebserverRawConfig]
+  {:request-header-max-size   (or (:request-header-max-size config)
+                                  default-request-header-size)
+   :so-linger-milliseconds    (so-linger-in-milliseconds config)
+   :idle-timeout-milliseconds (:idle-timeout-milliseconds config)})
```

Comment:
We've had this same debate on other PRs in the past, about other settings within Jetty.  I've weighed-in on this before, but since it's come up again ...

I'm with @cprice404 on this one.  To me, this boils down to the notion that Jetty knows better defaults for Jetty than we do.

I'd like to see us put this issue to bed for good ... either decide to continue coding our own defaults, and change this PR, or decide to stop coding our own defaults, leave this as-is, and come back around and remove all of the defaults we've already put in there.